app/controllers/assignments_controller.rb

```diff
@@ -570,7 +570,7 @@ def starter_file_group_file_data(starter_file_group)
       if (starter_file_group.path + file).directory?
         { key: "#{file}/" }
       else
-        { key: file, size: 1,
+        { key: file, size: 1, submitted_date: l(File.mtime(starter_file_group.path + file)),
```

Comment:
@JunruL Good change! But it looks like you need the `in_time_zone` call in order to make sure the time appears according to the current user's time zone. (See #5341 or talk to @TheXanMan321 about this)

BTW I tested this by going to the "Settings" tab and changing the current user's timezone.